server/json/q029.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "q029",
+  "edits": [
+    {
+      "lar": {"loanId": "00001"},
+      "stateCode": "36",
+      "countyCode": "021",
+      "censusTract": "0004.02",
+      "recommendedMSA": "00",
+      "verification": ""
```

Comment:
Hmm. I think it should default to false and you just make sure that the false radio isn't selected for the initial render (so people don't just breeze through the already selected buttons)